### DISCUSSION TITLE
MariaDB default username/password?

### DISCUSSION THREAD

**Discussion by level42ca** [community-scripts/ProxmoxVE#401] [Q&A] [answered]
Hey there, I just installed MariaDB using the default settings, but I didn't see anywhere that documented the default Username/Password. It's possible I missed something.

https://community-scripts.github.io/ProxmoxVE/scripts?id=mariadb

**Comment by MickLesk** [comment, marked as answer]
https://github.com/community-scripts/ProxmoxVE/discussions/192
